src/components/common/LinkButton.tsx

```diff
@@ -16,7 +16,7 @@ const LinkButton = (
     locale={locale}
     passHref
     style={{ pointerEvents: disabled ? 'none' : 'all' }}>
-    <Button ref={ref} disabled={disabled} {...props} />
+    <Button tabIndex={-1} ref={ref} disabled={disabled} {...props} />
```

Comment:
Can you elaborate more on this change as it affects the whole app, what's the intention behind the removal of the tab index?

Reply:
I found out a problem after putting the 'focus' styling that the `LinkButton` focuses twice, once on the `a` tag and once on the `button` which is unwanted behaviour I suppose. With this change the focus only goes on the `<a>` tag

Reply:
Thanks!